### PR TITLE
Fix circular progress indicator UI

### PR DESCRIPTION
**Pull request by Xanonymous-GitHub**
## Description 
The circular progress indicator in the course table page's appbar seems not pretty enough.
and sometimes it will cause UI overflow exception but not been noticed.

So this PR tried to fix these problems.

## How to Verify? 
Go to course table page, and see the circular progress indicator's color
check if you like it.

## Screenshots/GIF/Test Results (Optional) 

<table>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

File: lib/ui/other/my_progress_dialog.dart [hunk 1]
```diff
@@ -1,54 +1,41 @@
...
+                message == null
+                    ? const SizedBox.shrink()
+                    : Padding(
+                        padding: const EdgeInsets.only(top: 10),
+                        child: Text(
+                          message,
+                          style: const TextStyle(color: Colors.white),
+                          textAlign: TextAlign.center,
+                        ),
                       ),
```
Comment (ricky9667) [inline lib/ui/other/my_progress_dialog.dart:29]: [nit]
```suggestion
                if (message != null)
                    Padding(
                      padding: const EdgeInsets.only(top: 10),
                      child: Text(
                        message,
                        style: const TextStyle(color: Colors.white),
                        textAlign: TextAlign.center,
                      ),
                    ),
```

**Review by ricky9667 — APPROVED** [review]
LGTM with a nit comment.

File: lib/ui/other/my_progress_dialog.dart (hunk 1 above)
Comment (Xanonymous-GitHub) [reply lib/ui/other/my_progress_dialog.dart:29]: Oh ! 😭

Comment (ricky9667) [reply lib/ui/other/my_progress_dialog.dart]: Addressed at e438336.

**Pull request merged**